lib/authentication-context.js

```diff
@@ -203,16 +203,16 @@ AuthenticationContext.prototype._acquireToken = function(callback, tokenFunction
 };
 
 /**
- * Gets a token for a given resource.
- * @param {string}   resource                            A URI that identifies the resource for which the token is valid.
+ * Gets a token for given scope.
+ * @param {string}   scope                               A URI that identifies the scope for which the token is valid.
```

Comment:
Is this supposed to be array or string? Implementation and header are not matching here.